hesabu/parser.go

```diff
@@ -70,31 +71,31 @@ func (parsedEquations ParsedEquations) Solve() (map[string]interface{}, error) {
 	}
 
 	for _, key := range topsort {
+		expression, ok := parsedEquations.Equations[key]
+		if !ok {
+			// Should we already bail here?
+			// The solve won't work because we're missing a parameter.
+			log.Printf("[%s] was never defined", key)
+			continue
```

Comment:
shouldn't we stop there ?

Reply:
Think so to. At the moment we're just continuing on, and we will fail at Solving stage.

Might be better to already throw the error here (the only thing that kept me from implementing that immediately was that I'll need to accumulate the errors, otherwise you could end up in:

- a not defined
/me fixes a and runs again
- b not defined
/me fixes b and runs again
(at nauseum)

Reply:
I would fail fast...